morpho5/geometry/integrate.c

```diff
@@ -401,6 +401,343 @@ bool integrate_areaint(integrandfunction *function, unsigned int dim, double *x[
     return success;
 }
 
+/* **********************************************************************
+ * Volume integrals
+ * ********************************************************************** */
+
+// Nodes and weights from Journal of Computational and Applied Mathematics, 236, 17, 4348-4364 (2012)
```

Comment:
This journal article reference looks useful, and might be worth documenting in the `VolumeIntegral` help for the interested user.

Reply:
Good idea, but we may change the integration rule at short notice.